lib/vsc/install/commontest.py

```diff
@@ -37,34 +37,24 @@
 import logging
 import optparse
 import os
+import pkg_resources
 import pprint
 import re
 import sys
-import unittest
+from prospector.run import Prospector
+from prospector.config import ProspectorConfig
 
 from distutils import log
 from vsc.install.ci import JENKINSFILE, TOX_INI, gen_jenkinsfile, gen_tox_ini
 from vsc.install.headers import check_header
 from vsc.install.shared_setup import vsc_setup
 from vsc.install.testing import TestCase
 
-# No prospector in py26 or earlier
-# Also not enforced on installation
-HAS_PROSPECTOR = False
-Prospector = None
-ProspectorConfig = None
-
-try:
-    _old_basicconfig = logging.basicConfig
-    from prospector.run import Prospector
-    from prospector.config import ProspectorConfig
-    from prospector.__pkginfo__ import __version__ as prospector_version
-    HAS_PROSPECTOR = True
-    # restore in case pyroma is missing (see https://github.com/landscapeio/prospector/pull/156)
-    logging.basicConfig = _old_basicconfig
-except ImportError:
-    pass
+_old_basicconfig = logging.basicConfig
+# restore in case pyroma is missing (see https://github.com/landscapeio/prospector/pull/156)
```

Comment:
this link goes nowhere.

Reply:
did you mean https://github.com/PyCQA/prospector/pull/156 ? It is merged now. So maybe no longer needed?

Reply:
fixed in ca1c65d